app/models/team_user.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: team_users
+#
+#  id         :bigint           not null, primary key
+#  name       :string
+#  role       :integer
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#  team_id    :bigint           not null
+#
+# Indexes
+#
+#  index_team_users_on_team_id  (team_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (team_id => teams.id)
+#
+class TeamUser < ApplicationRecord
+  belongs_to :team, class_name: 'Team', optional: true
+
+  enum role: %i[team_employee master]
```

Comment:
as I see it's not following best practices

Reply:
deleted